src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements FigureArea {
```

Comment:
```suggestion
public class Circle extends Figure {
```
we can implement the FigureArea interface in the Figure class